src/react-apps/applications/runtime/src/types/global.ts

```diff
@@ -98,7 +98,9 @@ declare global {
     preselectedOptionIndex?: number;
   }
 
-  export interface IFormTextAreaComponent extends IFormComponent { }
+  export interface IFormTextAreaComponent extends IFormComponent {
+    readOnly: boolean;
```

Comment:
Can be put as an optional parameter for all components, as this is something that might be useful for other components in the future.

Reply:
Agreed, fixed.